conf/schema.xml

```diff
@@ -298,35 +306,33 @@
-   <!-- Solr must parse through a stored, indexed field to construct highlighted snippets around search terms.
-   The three options, termVectors, termPositions and termOffsets help Solr determine the locations in stored
-   fields of the search parameters, so it doesn't have to scan through a complete stored field each time snippets are requested.
+   <!-- Solr must parse through a stored, indexed field to construct highlighted snippets around search terms.  
+   The three options, termVectors, termPositions and termOffsets help Solr determine the locations in stored 
+   fields of the search parameters, so it doesn't have to scan through a complete stored field each time snippets are requested.  
    This is of particular concern with large full-text (OCR) fields. -->
    <dynamicField name="*_hlt" type="text"    indexed="true"  stored="true" termVectors="true" termPositions="true" termOffsets="true"/>
    <dynamicField name="*_b"  type="boolean" indexed="true"  stored="true"/>
    <dynamicField name="*_f"  type="sfloat"  indexed="true"  stored="true"/>
    <dynamicField name="*_d"  type="sdouble" indexed="true"  stored="true"/>
    <dynamicField name="*_dt" type="date"    indexed="true"  stored="true"/>
-   <dynamicField name="*_mdt" type="date"    indexed="true"  stored="true" multiValued="true"/>
    <dynamicField name="*_et" type="edgedText" indexed="true" stored="true" multiValued="true"/>
    <dynamicField name="*_mlt" type="mappedLowerText" indexed="true" stored="true" multiValued='false'/>
-   <!-- uncomment the following to ignore any fields that don't already match an existing
-        field name or dynamic field, rather than reporting them as an error.
-        alternately, change the type="ignored" to some other type e.g. "text" if you want
-        unknown fields indexed and/or stored by default -->
+   <!-- uncomment the following to ignore any fields that don't already match an existing 
+        field name or dynamic field, rather than reporting them as an error. 
+        alternately, change the type="ignored" to some other type e.g. "text" if you want 
+        unknown fields indexed and/or stored by default --> 
    <field name="dc.description" type="text_fgs"  indexed="true"  stored="true" multiValued="true"/>
    <dynamicField name="*" type="text_fgs"  indexed="true"  stored="true" multiValued="true"/>
-<field name="_version_" type="long" indexed="true" stored="true"/>
-
+   
  </fields>
 
- <!-- Field to use to determine and enforce document uniqueness.
+ <!-- Field to use to determine and enforce document uniqueness. 
       Unless this field is marked with required="false", it will be a required field
    -->
  <uniqueKey>PID</uniqueKey>
 
  <!-- field for the QueryParser to use when an explicit fieldname is absent -->
- <defaultSearchField>dc.description</defaultSearchField>
+ <defaultSearchField>catch_all_fields_mt</defaultSearchField>
```

Comment:
@ksclarke: You okay with us moving the default field to this as opposed to dc.description?